css/_highcontrast.scss

```diff
@@ -0,0 +1,263 @@
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * GLPI is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with GLPI. If not, see <http://www.gnu.org/licenses/>.
+ * ---------------------------------------------------------------------
+ */
+/****** TABLES ******/
+
+table {
+  box-sizing: border-box;
+  border-collapse: collapse;
+}
+
+.tab_cadre, .tab_cadrehov, .tab_cadre_fixehov {
+  border: 1px solid #ADADAD;
+}
+
+.tab_cadre_fixe {
+  border: 1px solid #ADADAD;
+  border-bottom: 0;
+}
+
+.tab_glpi {
+  th, td {
+    border: 0;
+  }
+}
+
+.tab_cadre {
+  th, td {
+    border-bottom: 1px solid #ADADAD;
+    border-top: 1px solid #ADADAD;
+    box-sizing: border-box;
+    border-collapse: collapse;
+  }
+}
+
+.tab_cadre_fixe {
+  th, td {
+    border-bottom: 1px solid #ADADAD;
+    border-top: 1px solid #ADADAD;
+    box-sizing: border-box;
+    border-collapse: collapse;
+  }
+}
+
+.tab_cadre_fixehov {
+  th, td {
+    border-bottom: 1px solid #ADADAD;
+    border-top: 1px solid #ADADAD;
+    box-sizing: border-box;
+    border-collapse: collapse;
+  }
+}
+
+.tab_cadrehov {
+  th, td {
+    border-bottom: 1px solid #ADADAD;
+    border-top: 1px solid #ADADAD;
+    box-sizing: border-box;
+    border-collapse: collapse;
+  }
+}
+
+.tab_cadre_postonly {
+  th, td {
+    border-bottom: 1px solid #ADADAD;
+    border-top: 1px solid #ADADAD;
+    box-sizing: border-box;
+    border-collapse: collapse;
+  }
+}
+
+.main_form tr.headerRow th {
+  border-bottom: 1px solid #ADADAD;
+  border-top: 1px solid #ADADAD;
+  box-sizing: border-box;
+  border-collapse: collapse;
+}
+
+/* Removes border on tables that have no tab_cadre class but nested in table with this class. */
+/* A border was set by previous rule if nested in previous selector. */
+
+table[class^="tab_cadre"] table:not([class^="tab_cadre"]) {
+  th, td {
+    border: 0;
+  }
+}
+
+#searchcriterias {
+  .tab_cadre_fixe {
+    border: 1px solid #ADADAD;
+  }
+  th, td {
+    border: 0;
+  }
+}
+
+.tab_bg_2 {
+  background-color: #E0E0E0;
+}
+
+.tab_bg_1_2 td, .tab_bg_2_2 td {
+  border-color: #7B3E3E;
+}
+
+.actor-head, .actor-content {
+  margin: 0;
+  min-height: 25px;
+  box-sizing: border-box;
+  border-collapse: collapse;
+}
+
+.tab_actors {
+  border-bottom: 1px solid #ADADAD;
+}
+
+.actor-head {
+  border-right: 1px solid #ADADAD;
+  border-bottom: 1px solid #ADADAD;
+}
+
+.actor_title {
+  border-right: 1px solid #ADADAD;
+}
+
+.order_DESC:before {
+  color: #000;
+}
+
+.tab_cadre_pager {
+  img {
+    &[src*=first\.png], &[src*=left\.png] {
```

Comment:
Values should be surrounded by `"` and the `\` is useless -> `&[src*="first.png"], &[src*="left.png"] {`.